research/flamby/fed_isic2019/mr_mtl_mkmmd/run_fold_experiment.slrm

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+#SBATCH --nodes=1
+#SBATCH --ntasks=1
+#SBATCH --ntasks-per-node=1
+#SBATCH --cpus-per-task=8
+#SBATCH --gres=gpu:1
+#SBATCH --mem=32G
+#SBATCH --partition=a40
+#SBATCH --qos=m
+#SBATCH --job-name=fl_five_fold_exp
+#SBATCH --output=%j_%x.out
+#SBATCH --error=%j_%x.err
+#SBATCH --time=12:00:00
+
+###############################################
+# Usage:
+#
+# sbatch research/flamby/fed_isic2019/mr_mtl_mkmmd/run_fold_experiment.slrm \
+#   path_to_config.yaml \
+#   path_to_folder_for_artifacts/ \
+#   path_to_folder_for_dataset/ \
+#   path_to_desired_venv/ \
+#   client_side_learning_rate_value \
+#   lambda value \
+#   mu value \
+#   l2 value \
+#   mkmmd_loss_depth value \
+#   server_address
+#
+# Example:
+# sbatch research/flamby/fed_isic2019/mr_mtl_mkmmd/run_fold_experiment.slrm \
+#   research/flamby/fed_isic2019/mr_mtl_mkmmd/config.yaml \
+#   research/flamby/fed_isic2019/mr_mtl_mkmmd//hp_results/ \
+#   /Users/david/Desktop/FLambyDatasets/fedisic2019/ \
+#   /h/demerson/vector_repositories/fl4health_env/ \
+#   0.0001 \
+#   0.01 \
+#   0.1 \
```

Comment:
It seems like perhaps mkmmd_loss_depth isn't present in this input?

Reply:
It is actually impressive how you detected this error 😅

Reply:
It's less impressive when I say that I just counted the arguments and realized we were 1 short 😂